components/admin/requests/doctor-information/Form.tsx

```diff
@@ -86,49 +38,42 @@ export default function DoctorInformationForm({
             {' (must be in British Columbia)'}
           </Box>
         </Text>
+        <Box paddingBottom="24px">
+          <TextField name="doctorInformation.addressLine1" label="Address line 1" required>
+            <FormHelperText color="text.secondary">
+              {'Street Address, P.O. Box, Company Name, c/o'}
+            </FormHelperText>
+          </TextField>
+        </Box>
 
-        <FormControl isRequired paddingBottom="24px">
-          <FormLabel>{'Address line 1'}</FormLabel>
-          <Input value={addressLine1} onChange={handleChange('addressLine1')} />
-          <FormHelperText color="text.secondary">
-            {'Street Address, P.O. Box, Company Name, c/o'}
-          </FormHelperText>
-        </FormControl>
-
-        <FormControl paddingBottom="24px">
-          <FormLabel>
-            {'Address line 2 '}
-            <Box as="span" textStyle="caption">
-              {'(optional)'}
-            </Box>
-          </FormLabel>
-          <Input value={addressLine2 || ''} onChange={handleChange('addressLine2')} />
-          <FormHelperText color="text.secondary">
-            {'Apartment, suite, unit, building, floor, etc'}
-          </FormHelperText>
-        </FormControl>
+        <Box paddingBottom="24px">
+          <TextField
+            name="doctorInformation.addressLine2"
+            label={
+              <>
+                {'Address line 2 '}
+                <Box as="span" textStyle="body-regular" fontSize="sm">
+                  {'(optional)'}
+                </Box>
+              </>
+            }
+          >
+            <FormHelperText color="text.secondary">
+              {'Apartment, suite, unit, building, floor, etc'}
+            </FormHelperText>
+          </TextField>
+        </Box>
 
         <Stack direction="row" spacing="20px">
-          <FormControl isRequired>
-            <FormLabel>{'City'}</FormLabel>
-            <Input
-              value={city}
-              onChange={event =>
-                onChange({
-                  ...doctorInformation,
-                  city: event.target.value,
-                })
-              }
-            />
-          </FormControl>
-
-          <FormControl isRequired>
-            <FormLabel>{'Postal code'}</FormLabel>
-            <Input value={postalCode} onChange={handleChange('postalCode')} />
+          <TextField name="doctorInformation.city" label="City" required />
+          <TextField name="doctorInformation.postalCode" label="Postal code" required>
             <FormHelperText color="text.secondary">{'Example: X0X 0X0'} </FormHelperText>
-          </FormControl>
+          </TextField>
         </Stack>
       </Box>
     </>
   );
 }
+
+// TODO: fix spacing between fields
+// TODO: get it to be savable???
```

Comment:
Are these TODO's resolved?

Reply:
Can you please delete these TODO comments if they're outdated?

Reply:
Yup fs my bad! I thought I made a commit getting rid of them but ig it didn't go through :/